diff -Nurp drakx-installer-images-1.48.orig//kernel/list_modules.pm drakx-installer-images-1.48//kernel/list_modules.pm
--- drakx-installer-images-1.48.orig//kernel/list_modules.pm	2010-02-28 05:54:31.000000000 +0200
+++ drakx-installer-images-1.48//kernel/list_modules.pm	2010-04-17 11:30:44.124445861 +0300
@@ -276,7 +276,8 @@ our %l = (
   {
     raid => [
       qw(dm-crypt dm-mirror dm-mod dm-zero linear lvm-mod multipath raid0 raid1 raid10 raid456 raid5 raid6),
-      qw(dm-delay dm-log dm-mem-cache dm-message dm-raid4-5 dm-region_hash faulty md-mod),
+      qw(dm-delay dm-log dm-memcache dm-message dm-raid45 dm-region-hash faulty md-mod),
+      qw(dm-emc dm-hp-sw dm-multipath dm-rdac dm-round-robin dm-snapshot),
     ],
     mouse => [
       qw(atixlmouse busmouse generic_serial inport ioc3_serial logibm logibusmouse msbusmouse pcips2 qpmouse synclinkmp),
